eservice/docs/test-scripts.md

```diff
@@ -176,17 +176,17 @@ extension.
 $ python test-request.py --no-ledger
 
 # Run the test with a ledger receiving transactions
-$ python test-request.py --ledger http://localhost:8008 --loglevel warn
+$ python test-request.py --ledger http://localhost:6600 --loglevel warn
 
```

Comment:
should we just replace these with ${PDO_LEDGER_URL}?

Reply:
done